b2share/config.py

```diff
@@ -458,3 +639,13 @@
 #: There is no password so don't send password change emails
 SECURITY_SEND_PASSWORD_CHANGE_EMAIL=False
 SECURITY_SEND_PASSWORD_RESET_NOTICE_EMAIL=False
+
+# Extra (Harry Kodden)
+# When testing in HTTP, both cookie secure and CSRF enforcement is switched off
+APP_ENABLE_SECURE_HEADERS = (os.environ.get('SESSION_COOKIE_SECURE', "True").upper() == "True".upper())
+
+SESSION_COOKIE_PATH="/"
+SESSION_COOKIE_SAMESITE="Lax"
+
+APP_THEME = ['semantic-ui']
+SECURITY_CHANGEABLE = False
```

Comment:
Allows changing of password for invenio accounts. Can be ignored since we are using B2ACCESS for user validation